src/plugins/stats.ts

```diff
@@ -31,8 +31,8 @@ module.exports = async robot => {
     return github.paginate(req, res => res.data)
   }
 
-  async function popularInstallations (installations) {
-    let popular = await Promise.all(installations.map(async installation => {
+  async function popularInstallations (installations): Promise<Array<PopularType>> {
```

Comment:
`PopularType[]` instead of `Array<PopularType>`
add a type for `installations`, and define that type as the type returned by the function `getInstallations`, if typescript gives you trouble in getInstallations, you can return the last line like this: `return github.paginate(req, res => res.data) as InstallationsType;` with `InstallationsType` the new type that you'll define (don't use that name, it's bad)

The `refresh` method should have types as well